.github/workflows/main.yml

```diff
@@ -31,6 +31,18 @@ jobs:
         operating-system: [ubuntu-latest, macos-latest]
     steps:
       - uses: actions/checkout@v2
+      - name: Cache local opam repository
+        uses: actions/cache@v2
+        with:
+          path: ~/.opam
+          key: ${{ runner.os }}-opam-4.11.0
+      - name: Set up opam
```

Comment:
I am a bit worried that we add `opam` as a dependency. I had problems with the stability of `opam` packages a few years ago, though this may have been improved since then. Are there alternatives, in case we experience a problem with `mdx`?

Reply:
My main hesitation with `cram` is that it appears to be abandoned: https://github.com/brodie/cram/issues/32 and it has not had a new release since 2016.

Reply:
I see. Then let's stay with `mdx` and merge!